src/test/java/org/apache/sysml/test/integration/functions/codegenalg/AlgorithmAutoEncoder.java

```diff
@@ -120,50 +128,54 @@ public void testAutoEncoder512SparseRewritesSpark() {
 	
 	private void runGLMTest(int batchsize, boolean sparse, boolean rewrites, ExecType instType)
```

Comment:
I known this issue existed before, but if it's not too much to ask, could you please change runGLMTest to runAutoEncoderTest because the wrong naming was simply a copy-paste bug.

Reply:
Thanks for your feedback. I will go through your suggestions and will do necessary changes.